### PR TITLE
MMT-3159: Upgrade rails to 6.0

### PR DESCRIPTION
**Pull request by dmistry1**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -74,7 +74,7 @@ class Application < Rails::Application
 
     # Store boolean values are in sqlite3 databases as 1 and 0 instead of 't' and
     # 'f' after migrating old data.
-    config.active_record.sqlite3.represent_boolean_as_integer = true
+    # config.active_record.sqlite3.represent_boolean_as_integer = true
```
Comment (jceaser) [inline config/application.rb:77]: is this still needed in some situations (like local development)? Why keep it?

File: db/schema.rb [hunk 2]
```diff
@@ -57,7 +54,7 @@
   end
 
   create_table "proposal_keyword_recommendations", force: :cascade do |t|
-    t.bigint "draft_proposal_id"
+    t.integer "draft_proposal_id"
```
Comment (jceaser) [inline db/schema.rb:57]: why go from 64 bit to 32?

File: config/application.rb (hunk 1 above)
Comment (dmistry1) [reply config/application.rb:77]: No it is not needed. I forgot to remove it, I kept it while testing thinking it was being used.

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -49,7 +49,7 @@
     # args: %w[headless disable-gpu no-sandbox --window-size=1920,1080 --enable-features=NetworkService,NetworkServiceInProcess]
   )
 
-  Capybara::Selenium::Driver.new(app, browser: :chrome, http_client: client, capabilities: capabilities, options: options)
+  Capybara::Selenium::Driver.new(app, browser: :chrome, http_client: client, options: options)
```
Comment (cgokey) [inline spec/rails_helper.rb:52]: Does capabilities not work anymore?

**Review by cgokey — APPROVED** [review]

**Review by tle1989 — APPROVED** [review]

**Pull request merged**